elrond-wasm/src/api/send_api.rs

```diff
@@ -76,6 +76,24 @@ where
 		}
 	}
 
+	/// Sends ESDT tokens to the target address. Handles any type of ESDT.
+	fn transfer_esdt(&self, token: &TokenIdentifier, nonce: u64, amount: &BigUint, to: &Address) {
```

Comment:
I would rather call this method transfer_tokens. You are handling both ESDTs and NFTs in the same method

Reply:
`transfer_tokens` is better, yes

Reply:
NFTs are ESDTs. transfer_tokens means any tokens, including EGLD. this only works with ESDT, hence the naming